internal/scylla/client.go

```diff
@@ -136,11 +136,65 @@ func (c *Client) doHttpRequest(req *http.Request) (resp *http.Response, temporar
 		return
 	}
 
-	temporaryErr = resp.StatusCode == http.StatusBadGateway || resp.StatusCode == http.StatusGatewayTimeout || resp.StatusCode == http.StatusTooManyRequests
+	body, err := getBody(resp)
+	if err != nil {
+		return nil, false, fmt.Errorf("failed to get response body: %w", err)
+	}
+
+	body2, err := getBody(resp)
+	if err != nil {
+		return nil, false, fmt.Errorf("failed to get response body: %w", err)
+	}
```

Comment:
Why do you read body twice? One would be enough.

Reply:
leftovers, removed